src/other-reprs.md

```diff
@@ -15,18 +15,26 @@ or C++. Any type you expect to pass through an FFI boundary should have
 necessary to soundly do more elaborate tricks with data layout such as
 reinterpreting values as a different type.
 
-However, the interaction with Rust's more exotic data layout features must be
+We strongly recommend using [rust-bindgen][] and/or [cbdingen][] to manage your FFI
+boundaries for you. The Rust team works closely with those projects to ensure
+that they work robustly and are compatible with current and future guarantees
+about type layouts and reprs.
+
+The interaction of `repr(C)` with Rust's more exotic data layout features must be
 kept in mind. Due to its dual purpose as "for FFI" and "for layout control",
 `repr(C)` can be applied to types that will be nonsensical or problematic if
 passed through the FFI boundary.
 
 * ZSTs are still zero-sized, even though this is not a standard behavior in
 C, and is explicitly contrary to the behavior of an empty type in C++, which
-still consumes a byte of space.
+says they should still consume a byte of space.
 
-* DST pointers (fat pointers), tuples, and enums with fields are not a concept
+* DST pointers (fat pointers) and tuples are not a concept
```

Comment:
I would prefer if you said either `thick pointers` or `fat/thick pointers`. I'm, at least personally, attempting to change this language right now.

Reply:
i also have been using "double pointer" instead

Reply:
There are no two pointers in a slice though, so "double" seems like a bad choice to me.

I don't know why you want to change language here, but shouldn't that also be changed in the [API docs then](https://doc.rust-lang.org/nightly/std/ops/trait.CoerceUnsized.html)? Also I am concerned about understandability, "fat pointer" is a standard term (even used in academia for "pointers annotated with stuff") that many will have heard before, using a different term puts up an additional unnecessary barrier. But more generally I wonder if there has been discussion about this where I could read up on your reasoning?

Reply:
short for "double-width pointer"; all the current ones are two usizes, even if the second isn't a pointer itself.